tests/dummy/app/templates/components/test-component.hbs

```diff
@@ -27,3 +27,17 @@
              propertyPath="textArea"
              updated=(action "update")
              disabled=disabled}}
+<br/>
+
+{{form-field type="paper-input"
+             object=object
+             propertyPath="text"
+             updated=(action "update")
+             disabled=true
+             label="lel"}}
+<br/>
+
+
+<div class="layout layout-sm-column">
```

Comment:
Why did you add this div and it's content. It's a direct paper input. Nothing to do with our form-fields right?